librz/core/cfile.c

```diff
@@ -561,16 +529,14 @@ CURSOR cursor position(offset from curseek)
 	sdb_sync(config_sdb);
 	sdb_free(config_sdb);
 	rz_sys_setenv("RZ_CONFIG", config_sdb_path);
-
 	rz_sys_setenv("RZ_BIN_LANG", rz_config_get(core->config, "bin.lang"));
 	rz_sys_setenv("RZ_BIN_DEMANGLE", rz_config_get(core->config, "bin.demangle"));
 	rz_sys_setenv("RZ_ARCH", rz_config_get(core->config, "asm.arch"));
 	rz_sys_setenv("RZ_BITS", sdb_fmt("%" PFMT64u, rz_config_get_i(core->config, "asm.bits")));
-	rz_sys_setenv("RZ_COLOR", rz_config_get_i(core->config, "scr.color") ? "1" : "0");
-	rz_sys_setenv("RZ_DEBUG", rz_config_get_b(core->config, "cfg.debug") ? "1" : "0");
-	rz_sys_setenv("RZ_IOVA", rz_config_get_i(core->config, "io.va") ? "1" : "0");
+	rz_sys_setenv("RZ_COLOR", sdb_fmt("%" PFMT64u, rz_config_get_i(core->config, "scr.color")));
```

Comment:
`rz_strf` please.

Reply:
i'll reset this back to the original code and even maybe remove this.